x-pack/plugins/security_solution/public/common/lib/telemetry/constants.ts

```diff
@@ -77,6 +77,11 @@ export enum TelemetryEventTypes {
   OnboardingHubStepOpen = 'Onboarding Hub Step Open',
   OnboardingHubStepFinished = 'Onboarding Hub Step Finished',
   OnboardingHubStepLinkClicked = 'Onboarding Hub Step Link Clicked',
+  ManualRuleRunOpenModal = 'Manual Rule Run Open Modal',
+  ManualRuleRunExecute = 'Manual Rule Run Execute',
+  ManualRuleRunCancelJob = 'Manual Rule Cancel Job',
```

Comment:
Without "Run" this description is sorta ambiguous (or less precise than it could be): 

```suggestion
  ManualRuleRunCancelJob = 'Manual Rule Run Cancel Job',
```